flask_request_validator/exceptions.py

```diff
@@ -45,7 +45,7 @@ def __str__(self):
         return 'Invalid request header. ' + json.dumps(self.errors)
 
 
-class TooMuchArguments(Exception):
+class TooManyArguments(Exception):
```

Comment:
Hello. Thank you for your work!
I think I'll increase `patch` version. But someone can handle `TooMuchArguments` using `v3.0.0`. 

Can you leave 2 errors for now for compatibility? Just an example:

```
class TooManyArguments(Exception):
     pass

class TooMuchArguments(TooManyArguments):
    """
    .. deprecated:: 3.0.1
          Use TooManyArguments
    """
```

Reply:
You're right. I totally forgot about that. In https://github.com/d-ganchar/flask_request_validator/pull/51/commits/47d24f106b796a04a00f2ed85c75380abca7cd22 I added an alias for that. Hope that fine.

I also plan to make more changes like:
- added more useful rules
- split the tests into multiple files and refactor them
- refactor the validator

Please let me know if you are okay with this suggestions. If you have any wishes or concerns feel free to tell :)